Net24Solution/Net24Task/MonitoringApp.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Dynamic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using NLog;
+
+namespace Net24Task
+{
+    public class MonitoringApp
```

Comment:
MonitoringApp and MonitoringAppConfigBuild do the same functionality. 
Keep only MonitoringApp